static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.js

```diff
@@ -221,12 +234,12 @@ const requestBids = (
         return requestQueue;
     }
 
-    const adUnits: Array<PrebidAdUnit> = getHeaderBiddingAdSlots(
-        advert,
-        slotFlatMap
-    )
-        .map(slot => new PrebidAdUnit(advert, slot))
-        .filter(adUnit => !adUnit.isEmpty());
+    const adUnits: Array<PrebidAdUnit> = (await Promise.all(
+        getHeaderBiddingAdSlots(advert, slotFlatMap).map(
+            // eslint-disable-next-line new-cap
+            async slot => new PrebidAdUnit.build(advert, slot)
```

Comment:
The lowercase “B” in `build` makes `eslint` unhappy. Should we consider using `PrebidAdUnit.Build`?